WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginGoogleFragment.java

```diff
@@ -313,16 +320,24 @@ public void onSocialChanged(OnSocialChanged event) {
         // Response returns error for non-existing account and existing account not connected.
         if (event.isError()) {
             AppLog.e(T.API, "LoginEmailFragment.onSocialChanged: " + event.error.type + " - " + event.error.message);
-            AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_FAILED, event.getClass().getSimpleName(),
-                    event.error.type.toString(), event.error.message);
+
+            if (event.error.type != AccountStore.AccountSocialErrorType.USER_EXISTS) {
```

Comment:
The `import org.wordpress.android.fluxc.store.AccountStore;` statement needs to be added to be able to build with this conditional.

Reply:
How this could have been omitted baffles me.  Adding back.